routing/payment_session.go

```diff
@@ -14,6 +14,22 @@ import (
 // to prevent an HTLC being failed if some blocks are mined while it's in-flight.
 const BlockPadding uint16 = 3
 
+// ValidateCLTVLimit is a helper function that validates that the cltv limit is
+// less than the final cltv delta parameter, optionally including the
```

Comment:
nit: I think what you mean here is that we want to validate the cltv limit is *greater* than the final cltv delta?

Reply:
Changed comment